pkgs/development/python-modules/ngs-tools/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  joblib,
+  numba,
+  numpy,
+  pysam,
+  shortuuid,
+  tqdm,
+  typing-extensions,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "ngs-tools";
+  version = "1.8.5";
+  format = "setuptools";
```

```suggestion
	pyproject = true;
```
Please add `pyproject = true;` and `build-system = [ setuptools ];` to avoid the legacy setuptools build.

> Note that the pyproject format falls back to using setuptools, so you can use pyproject = true even if the package only has a setup.py. When set to false, you can use the existing [hooks](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#setup-hooks) or provide your own logic to build the package. This can be useful for packages that don't support the pyproject format. When unset, the legacy setuptools hooks are used for backwards compatibility.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters